client/src/types/elementType.js

```diff
@@ -6,8 +6,8 @@ const elementType = PropTypes.shape({
   title: PropTypes.string,
   blockSchema: PropTypes.object,
   inlineEditable: PropTypes.bool,
-  published: PropTypes.bool,
-  liveVersion: PropTypes.bool,
+  isPublished: PropTypes.bool,
+  isLiveVersion: PropTypes.bool,
```

Comment:
Unrelated refactoring - I noticed these were wrong when I tried to use the old keys but they kept showing `undefined`